spot_wrapper/spot_arm.py

```diff
@@ -500,6 +500,50 @@ def hand_pose(self, data) -> typing.Tuple[bool, str]:
 
         return True, "Moved arm successfully"
 
+    def handle_arm_velocity(
+        self, arm_velocity_command: arm_command_pb2.ArmVelocityCommand.Request, cmd_duration: float = 0.15
+    ) -> typing.Tuple[bool, str]:
+        """
+        Set the velocity of the arm TCP
+
+        Args:
+            arm_velocity_command: Protobuf message to set the arm velocity
+            cmd_duration: (optional) Time-to-live for the command in seconds.
+
+        Returns:
+            Boolean success, string message
+        """
+
+        try:
+            success, msg = self.ensure_arm_power_and_stand()
+            if not success:
+                self._logger.info(msg)
+                return False, msg
+            else:
+                end_time = self._robot.time_sync.robot_timestamp_from_local_secs(time.time() + cmd_duration)
+
+                arm_velocity_command2 = arm_command_pb2.ArmVelocityCommand.Request(
+                    cylindrical_velocity=arm_velocity_command.cylindrical_velocity,
+                    angular_velocity_of_hand_rt_odom_in_hand=arm_velocity_command.angular_velocity_of_hand_rt_odom_in_hand,
+                    cartesian_velocity=arm_velocity_command.cartesian_velocity,
+                    end_time=end_time,
+                )
```

Comment:
Why do you copy the request to `arm_velocity_command2` here?

Reply:
oh now I see that you modified the end time.
Could you not do something like this?
```Python
new_end_time = . . .
arm_velocity_command.end_time = new_end_time
```